src/traits.rs

```diff
@@ -233,6 +233,33 @@ pub trait TagExt: Accessor + Into<Tag> + Sized {
 	fn clear(&mut self);
 }
 
+/// Split and rejoin tags.
+///
+/// Useful and required for implementing read/modify/write round trips.
+pub trait SplitAndRejoinTag {
```

Comment:
Can you add an `# Examples` section to the methods? I try to do that with all new public methods.

Reply:
I already thouught about that when adding my code example here. Was just lazy ;)

Reply:
Since you always apply these operations in pairs the example should be added to the trait.